### ISSUE TITLE
[BUG] - NOSE GEAR RIM AND TIRE MODELLING ALL ACFT TYPES

### ISSUE THREAD

**Issue by PANAM741** [invernyx/md11-bugs#1425]
### Describe the bug

The nose gear rim modelling is not the same of the real thing. Either the tire shape and size

### Expected behaviour

Please reshape the rim and fix the tire. Also adjust the rim size.

first pic is the tfdi MD11

### Evidence

![IMG_5032](https://github.com/invernyx/md11-bugs/assets/71863590/30e51418-6c82-4ed5-9c3a-338d3d02075c)
![IMG_5033](https://github.com/invernyx/md11-bugs/assets/71863590/fdd25a5b-ab69-4426-a05d-b7d46c3209b1)
![IMG_5038](https://github.com/invernyx/md11-bugs/assets/71863590/0cb8122e-9af7-4041-9bd2-e44cac95f543)
![IMG_5039](https://github.com/invernyx/md11-bugs/assets/71863590/00923f02-00e4-4962-a1cc-a838132689c1)

![IMG_5040](https://github.com/invernyx/md11-bugs/assets/71863590/18446c1b-7091-45e6-9290-beaf56d0589d)

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Aircraft Type

Passenger GE

### Agreement

- [X] Yes

**Comment by TheFightingRaven** [comment]
![image](https://github.com/invernyx/md11-bugs/assets/138527559/3a3064d8-0baa-4650-8c0e-68868650745b)
this is also an md11 nosegear, multiple wheels exist, model will not be changed

**Comment by PANAM741** [comment]
Yeah, if you see in your pic, the model isn’t closer to the one on the TFDI; I still consider the rim model and size should be reshaped, looks too different to the one on the real thing. Please compare carefully and you’ll see what i’m talking about.
